WatchApp Extension/Controllers/ActionHUDController.swift

```diff
@@ -68,6 +68,23 @@ final class ActionHUDController: HUDInterfaceController {
         } else if overrideButtonGroup.state == .disabled {
             overrideButtonGroup.state = .off
         }
+        
+        requestContextUpdate()
```

Comment:
Should some equivalent of `requestContextUpdate` also be invoked in ChartHUDController's `update`, in case the user leaves that screen open rather than ActionHUDController?